samples/python/resize_pil.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+'''
+Compare resize function with interpolations: cv.INTER_NEAREST, cv.INTER_NEAREST_PIL, PIL.Image.NEAREST
+
+Usage:
+  resize_pil.py [<image>]
+
+'''
+
+# Python 2/3 compatibility
+from __future__ import print_function
+
+import numpy as np
+import cv2 as cv
+
+import sys
+
+from PIL import Image
+import argparse
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("fname", nargs='?', help="picture filename to test resize", default='lena.jpg')
+    args = parser.parse_args()
+    fn = args.fname
+
+    img_cv_lena = cv.imread(cv.samples.findFile(fn), cv.IMREAD_GRAYSCALE)
```

Comment:
as for me, `_lena` suffix is extra. What if user have chosen different image? Looks confusing.

Reply:
Yes, I've removed it

Reply:
Dear Colleagues, any questions\suggestions so far?